File: src/main/java/com/ort/app/web/controller/AboutController.java
package com.ort.app.web.controller;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class AboutController {

    // ===================================================================================
    //                                                                             Execute
    //                                                                             =======
    @GetMapping("/about")
    private String about(Model model) {
        return "about";
    }

    @GetMapping("/announce")
    private String announce(Model model) {
        return "announce";
    }

    @GetMapping("/rule")
    private String rule(Model model) {
        return "rule";
    }

    @GetMapping("/intro")
    private String intro(Model model) {
        return "intro";
    }

    @GetMapping("/practice")
    private String practice(Model model) {
        return "practice";
    }
}

File: src/main/kotlin/com/ort/app/api/AboutController.kt
package com.ort.app.api

import com.ort.app.api.view.RuleContent
import com.ort.app.domain.model.skill.Skills
import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping

@Controller
class AboutController {

    @GetMapping("/about")
    fun about(): String = "about"

    @GetMapping("/announce")
    fun announce(): String = "announce"

    @GetMapping("/rule")
    fun rule(model: Model): String {
        model.addAttribute("content", RuleContent())
        return "rule"
    }

    @GetMapping("/intro")
    fun intro(): String = "intro"

    @GetMapping("/practice")
    fun practice(): String = "practice"

    @GetMapping("/faq")
    fun faq(): String = "faq"
}